.devcontainer/Dockerfile

```
@@ -78,18 +82,29 @@ RUN python3 -m pip install \
 # Set zsh as default shell
 SHELL ["/bin/zsh", "-c"]
 
-# Create home directory and colcon workspace
-RUN mkdir -p "/root/colcon_ws"
+# Remove the users group, because when it exists on the host system
+# the devcontainer will not dynamically update the containerUser GID,
+# when the host user is part of the users group.
+# Then create a bitbots user with home directory and add allow it to use sudo
+RUN groupdel users \
```

Hadolint

[error] ShellCheck only supports sh/bash/dash/ksh scripts. Sorry!

(SC1071)